backstop/tests/carousel.html

```diff
@@ -0,0 +1,471 @@
+      [data-slide-id="1"] { background: linear-gradient(to right, #2193b0, #6dd5ed); }
+      [data-slide-id="2"] { background: linear-gradient(to right, #cc2b5e, #753a88); }
+      [data-slide-id="3"] { background: linear-gradient(to right, #00467f, #a5cc82); }
+      [data-slide-id="4"] { background: linear-gradient(to right, #ec008c, #fc6767); }
+      [data-slide-id="5"] { background: linear-gradient(to right, #1488cc, #2b32b2); }
+      [data-slide-id="6"] { background: linear-gradient(to right, #bbd2c5, #536976); }
+      [data-slide-id="7"] { background: linear-gradient(to right, #9796f0, #fbc7d4); }
+      [data-slide-id="8"] { background: linear-gradient(to right, #b79891, #94716b); }
+      [data-slide-id="9"] { background: linear-gradient(to right, #acb6e5, #86fde8); }
+      [data-slide-id="10"] { background: linear-gradient(to right, #ffe000, #799f0c); }
+
+    </style>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/chevron-left"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/chevron-right"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/pause"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/play"
+    ></script>
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <h1>Carousel</h1>
+    <hr />
+
+    <h2>Default</h2>
+    <section id="demo-default">
+
+      <div
+        class="iui-carousel demo-carousel-width"
+        aria-roledescription="carousel"
+        aria-label="Introduction to iTwinUI"
+        aria-live="off"
+        aria-atomic="true"
+      >
+        <nav class="iui-carousel-navigation">
+          <div class="iui-carousel-navigation-left">
+            <button
+              class="iui-button iui-borderless iui-small iui-carousel-navigation-animation-button"
+              type="button"
+              aria-label="Pause animation"
+              aria-pressed="true"
+              data-action="play"
+            >
+              <svg-pause
+                class="iui-button-icon"
+                aria-hidden="true"
+              ></svg-pause>
+            </button>
+            <button
+              class="iui-button iui-borderless iui-small iui-carousel-navigation-button"
+              type="button"
+              aria-controls="myCarousel-items"
+              aria-label="Previous slide"
+              tabindex="-1"
+            >
+              <svg-chevron-left
+                class="iui-button-icon"
+                aria-hidden="true"
+              ></svg-chevron-left>
+            </button>
+          </div>
+
+          <ol
+            class="iui-carousel-navigation-dots"
+            role="tablist"
+            aria-controls="myCarousel-items"
```

Comment:
This `aria-controls` also looks wrong because the list itself is not controlling anything. Though I'm less sure about it because we made it focusable and decided to add keyboard handling

Reply:
This one I am unsure about.  [w3](https://www.w3.org/TR/wai-aria-practices-1.2/examples/carousel/carousel-2-tablist.html) does not have it present on theirs, so I guess I'll remove it.